src/main/java/uk/gov/pay/api/model/search/card/SearchRefundsResults.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.pay.api.model.search.card;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
so many intermediate objects :( not your fault, but it's growing a bit out of control. We might want to address this, but not as part of this story, I guess